nvme-print.c

```diff
@@ -2292,6 +2292,149 @@ static void json_supported_cap_config_log(
 	json_free_object(root);
 }
 
+void nvme_show_fdp_config_fdpa(uint8_t fdpa)
+{
+	__u8 valid = (fdpa >> 7) & 0x1;
+	__u8 rsvd = (fdpa >> 5) >> 0x3;
+	__u8 fdpvwc = (fdpa >> 4) & 0x1;
+	__u8 rgif = fdpa & 0xf;
+
+	printf("  [7:7] : %#x\tFDP Configuration %sValid\n",
+		valid, valid ? "" : "Not ");
+	if (rsvd)
+		printf("  [6:5] : %#x\tReserved\n", rsvd);
+	printf("  [4:4] : %#x\tFDP Volatile Write Cache %sPresent\n",
+		fdpvwc, fdpvwc ? "" : "Not ");
+	printf("  [3:0] : %#x\tReclaim Group Identifier Format\n", rgif);
+}
```

Comment:
I can find the corresponding JSON output for this? Do I miss something?

Reply:
Generally I don't think nvme-cli does this for the json output (this is individual bits in a "flag" value).